package.json

```diff
@@ -2,10 +2,6 @@
   "name": "mobify-code-style",
   "version": "2.4.3",
   "description": "Code style guide and linting tools for Mobify",
-  "main": "index.js",
-  "scripts": {
-    "test": "grunt test"
```

Comment:
Weird. I can't even find a `Gruntfile` in this repo, so this doesn't appear to have done anything.